appeals/web/testing/util/testHelpers.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-env jest */
+
+/**
+ * Asserts that the html snapshot does not contain an error message and matches the snapshot
+ * @param {string} htmlOutput
+ * @param {string} errorMessage
+ */
+export function toMatchSnapshotWithoutError(
+	htmlOutput,
+	errorMessage = '<h2 class="govuk-error-summary__title"> There is a problem</h2>'
```

Comment:
Would it be better to do a slightly relaxed check, like just `There is a problem`?